app/features/dm.ts

```diff
@@ -180,6 +126,21 @@ export class DirectedMessageController implements DirectMessageGetter {
         return messages;
     }
 
+    public getLastestMessage(): ChatMessage | undefined {
```

Comment:
You can use a reduce function

Reply:
There is no need to check the chat messages' participants because it's in this class already

Reply:
ok, i found it in the `whoIamTalkingTo` func